lib/hanami/view/template.rb

```diff
@@ -7,9 +7,17 @@ module View
     # @since 0.1.0
     class Template
       def initialize(template, encoding = Encoding::UTF_8)
-        # NOTE disable_escape: true is for Slim compatibility
-        # See https://github.com/hanami/assets/issues/36
-        @_template = Tilt.new(template, nil, default_encoding: encoding, disable_escape: true)
+        default_options = {default_encoding: encoding}
+
+        options = default_options
+
+        if Tilt[template] != Tilt::HamlTemplate
```

Comment:
This is conceptually wrong: why all the other templates need that `disable_escape` option if it's only for Slim?
Shouldn't it be `if Tilt[template] == Slim::Template`?

Reply:
Right, that's another possibility. I just wanted to keep it as close to what we had before as possible...